site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+Texture Dumping is a way to gather all textures from a game whilst playing through it. No need to do anything fancy, all you need to do is play! These can be used for archival purposes or to create your own Custom Texture Packs to add right back into Citra.
+
+First [added to Citra Desktop](https://github.com/citra-emu/citra/pull/4868) all the way back in November 2019, first time developer [LevithaninWaves](https://github.com/LeviathaninWaves) tasked themselves with implementing this feature over on our Android build.
+
+To add Custom Textures to Citra, navigate to `Internal Storage -> citra-emu`, create a folder called `load`, another folder inside that called `textures` then a folder named with the TitleID of your game. This can be found on the [3DS releases database](http://3dsdb.com/) if you do not know it. 
+Place your custom textures inside this folder with the TitleID of your game. Dumped textures are stored in  `Internal Storage -> citra-emu -> dump -> textures`.
+Both Dump Textures and Custom Textures can be enabled in `Settings -> Graphics -> Utility`
+
+{{< figure src="textures.png"
+    title="Games can now be viewed so much clearer thanks to this!" >}}
+
+### Add internet permission to android manifest ([#6167](https://github.com/citra-emu/citra/pull/6167)) by [PabloMK7](https://github.com/PabloMK7)
+
+This addition is basically what it says on the tin. Some homebrew apps require internet access to download data needed for them to work correctly, such as the Mario Kart 7 modpack [CTGP-7](https://ctgp-7.github.io/). As this is what is known as a “normal permission” granted from Android, any users of Citra do not need to agree to any additional privacy policies that may have been introduced due to this.
+
+NOTE: This PR does not mean that Citra Android has multiplayer access. This is only related to homebrew apps at the moment.
+
+### Allow opening in-emulation menu by pressing Back ([#6248](https://github.com/citra-emu/citra/pull/6248)) by [JosJuice](https://github.com/JosJuice)
+
+With the Android 13 update released on some devices came a big issue with Citra. No longer could you swipe down from the top to open the in-game menu! Instead, just the phone notification bar would be brought down.
+
+Due to this, you could no longer access Save States, Amiibo, Screen Layout, Overlay, FPS counter and so many more vital options to use Citra! This was a massive problem!
+
+The solution was to just use the back button found on all devices to open the in-emulation menu instead! This can be accessed from the bottom of the device, next to the home button and recents button. By doing this, the in-emulation menu can be accessible on a number of devices now, such as Chromebooks running Citra Android and VR headsets, not just devices running Android 13.
```

Comment:
```suggestion
The solution was to just use the back button found on all devices to open the in-emulation menu instead! This can be accessed from the bottom of the device, next to the home button and recents button. The in-emulation menu is accessible on a number of devices now, such as Chromebooks running Citra Android and VR headsets, not just devices running Android 13.
```